include/SDL_syswm.h

```diff
@@ -289,7 +289,6 @@ struct SDL_SysWMinfo
         {
             struct wl_display *display;             /**< Wayland display */
             struct wl_surface *surface;             /**< Wayland surface */
-            struct wl_shell_surface *shell_surface; /**< Wayland shell_surface (window manager handle) */
```

Comment:
I think this has to still be here, but as a void*, so something like

```
void *shell_surface; /** Used to be wl_shell_surface, no longer used */
```

But I could be wrong here... not sure we've ever _removed_ a member before. @icculus?

Reply:
It has to stay and be NULL (but may change to void*); we can add a new field after it if we want to expose a different interface, since these should be checking a version field before filling in the struct.

Reply:
This line still needs to be

```
void *shell_surface; /** Used to be wl_shell_surface, no longer used */
```